CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.0.3
+
+- Updated node-notifier version from `^5.1.2` to `^6.0.0`
```

Comment:
Could you tweak the comment so it links back to the PR and thank yourself too please? 😄

Reply:
@johnnyreilly done! Thanks for the comment.